Console/Command/QueueShell.php

```diff
@@ -11,7 +11,9 @@
 
 App::uses('Folder', 'Utility');
 App::uses('File', 'Utility');
-App::import('Vendor', 'Hipchat', array('file' => 'HipchatPhp/src/HipChat/HipChat.php')); 
+if (!class_exists('HipChat\HipChat')) {
```

Comment:
It is a bit of a hack.  The Queue Shell includes Hipchat once when it is used to run the worker, new since adding support for the queue status monitor.

The kudos message task in the main repo also includes HipChat, which causes the error.  The Cake autoloader doesn't handle it with the fancier 'uses' method because namespaces and 2 classes in the same file.  This seemed like the best way to handle it for now.